source/assets/scripts/main.js

```diff
@@ -1,11 +1,119 @@
+    addCollapsibleControls();
+    addTasks();
+    let mondayFunc = document.getElementById("mFunc");
+    mondayFunc.innerHTML+=`
+    function deleteTask(t){
+        let id = t.id.substring(1,t.id.length);
+        let day = t.parentNode;
+        day.remove();
     }
-    else{
-        taskBoard.style.display = "none";
+    function editTask(t){
+        let id = t.id.substring(1,t.id.length);
+        let day = t.parentNode;
+        // inputRef[0] is checkbox, inputRef[1] is input text box.
+        let inputRef = day.getElementsByTagName('input')[1];
+        inputRef.disabled = false;
+        let confirmBut = day.getElementsByTagName('button')[0];
+        confirmBut.disabled = false;
     }
-});
\ No newline at end of file
+    function confirmButton(t){
+        let id = t.id.substring(1,t.id.length);
+        let day =t.parentNode;
+        // inputRef[0] is checkbox, inputRef[1] is input text box.
+        let inputRef = day.getElementsByTagName("input")[1];
+        // t.remove();
+        t.disabled = true;
+        inputRef.disabled = true;
+    }
+    `
+}
+/**
+ * Add collapsible controls to the element with collapsible class.
+ * When user click on element, hidden sibling text element will be shown,
+ * or expanded text element will be hidden. 
+ */
+function addCollapsibleControls(){
+    // find all collapsible control element
+    let lists = document.getElementsByClassName("collapsible");
+    // when the control element is clicked 
+    Array.from(lists).forEach(dailyList => {
+        dailyList.addEventListener("click", (event) => {
+            // find corresponding sibling text element
+            let taskBoard = dailyList.nextElementSibling;
+            // when task board is hidden, show the text board
+            if (taskBoard.classList.contains("shrink")){
+                taskBoard.classList.remove("shrink");
+                taskBoard.classList.add("expand");
+            }
+            // when text board is shown, shrink the text board
+            else{
+                taskBoard.classList.add("shrink");
+                taskBoard.classList.remove("expand");
+            }
+        });
+    });
+
+}
+/**
+ * add new tasks
+ * @param None
+ * @return new div block in week days.
+ */
+function addTasks(){
+    // add new task to each week days
+    // let elementId = new Map();
+    let daysDiv = document.getElementById("weekdays").getElementsByTagName("div");
+    // add (plus button id, taskcount id) pair to dayToTask.
+    let dayToTask = new Map();
+    for(const day of daysDiv){
+        dayToTask.set("addButton" + day.id, day.id+"Id")
+    };
+    // add event listener to all days
+    for(const [key,value] of dayToTask){
+        document.getElementById(key).addEventListener('click', function(){createNewTask(value,key.substring(9,key.length))}, false);
+    }
+    
+    /**
+     * EventListener function to add new task everytime click add button.
+     * @param string taskCount and dayOfWeek
```

Comment:
if there is two param, you should keep the second param in the next line. Like @param string taskCount
        @param  string dayOfWeek